bin/generate.py

```diff
@@ -636,8 +636,7 @@ def up_to_date():
                 return
 
         # Generate .ans and .interaction if needed.
-        # TODO: Disable this with a flag.
-        if not (testcase.bad_input or testcase.bad_output):
+        if not (testcase.bad_input or testcase.bad_output or config.args.skip_solution):
```

Comment:
for symmetry with the other comment this should be
```suggestion
        if not config.args.skip_solution and not (testcase.bad_input or testcase.bad_output):
```
but it's quite ugly so maybe just `if not (config.args.skip_solution or ...)` instead